### PR TITLE
I18n phase one report - GSoC'2021

### PR DESCRIPTION
**Pull request by pavanjoshi914**
Phase one GSoC'2021 Report for I18n :grinning: 

Deploy Preview:: https://deploy-preview-79--cvblog.netlify.app/posts/2021/i18n-phase-one-report/

### CODE REVIEW

**Review by ShreyaPrasad1209 — APPROVED** [review]
Wow, I really like the way you presented your work 🚀

File: content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+* [Research based documentation for I18n](https://gist.github.com/pavanjoshi914)
+* [Curated list for all I18n Pull Requests](https://github.com/CircuitVerse/CircuitVerse/issues/2288)
+* [Documentation](https://github.com/CircuitVerse/CircuitVerse/wiki/Internationalization-%28I18n%29)
+* [Module specific I18n Rules](https://github.com/CircuitVerse/CircuitVerse/wiki/I18n-Localization-Rules-%28Rails%29)
+ 
+&nbsp;
+ 
+### Additional Information
+ 
+All the commitments I have made for phase-1 of the coding period had been achieved. Instead of CircuitVerse Docs, I have started my work with Mobile App parallely and I18n for Docs will be done in the ending weeks as it becomes ready for I18n.
```
Comment (ShreyaPrasad1209) [inline content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md:109]: Replace parallely with "in parallel"

File: content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ 
+&nbsp;
+ 
+### Blog Posts
+ 
+I have also created my own blog covering my GSoC journey as well as my preparation during pre-GSoC, in order that future GSoC aspirants also get benefitted from it .
```
Comment (ShreyaPrasad1209) [inline content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md:115]: Typo: Replace benefitted with benefited

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@pavanjoshi914 everything looks great except for few minor typos. Fix that and then we will merge your PR! 🎉

**Comment by pavanjoshi914** [comment]
> Wow, I really like the way you presented your work

Thanks, @ShreyaPrasad1209 :tada: :tada:

File: content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md (hunk 1 above)
Comment (pavanjoshi914) [reply content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md]: corrected

File: content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md (hunk 2 above)
Comment (pavanjoshi914) [reply content/posts/Pavan_i18n_phase_one_report_gsoc_2021.md]: replaced with  ``` so that future GSoC aspirants can receive an advantage from it . ```

**Comment by pavanjoshi914** [comment]
@ShreyaPrasad1209 did the required changes!

**Review by ShreyaPrasad1209 — APPROVED** [review]
Awesome 🎉 I’ll merge it

**Pull request merged**